documentation/modules/appendix_faq.adoc

```diff
@@ -50,7 +50,7 @@ A `Node` is a cluster-scoped resource, so access to it can only be granted throu
 By default, standard Kubernetes users will not have the privileges necessary to manage the custom resources handled by the Cluster Operator.
 The cluster administrator can grant a user the necessary privileges using Kubernetes RBAC resources.
 
-For more information, see xref:adding-users-the-strimzi-admin-role-{context}[Strimzi Administrators].
+For more information, see link:{BookURLDeploying}#adding-users-the-strimzi-admin-role-{context}[Designating Strimzi administrators^].
```

Comment:
is it fine to use `-{context}` instead of `-str` in the URL query?

Reply:
It's okay. Primary context is _str_ in each guide. I just swapped what we were using already.  I'd like to reduce the number of links between guides ultimately.